Cargo.toml

```diff
@@ -15,12 +15,13 @@ required-features = ["openai-vocabulary-file"]
 
 [dependencies]
 ahash = "0.8.6"
+ndarray = { version = "0.15.6", optional = true }
 regex = "1.10.2"
 
 [dev-dependencies]
 criterion = "0.5.1"
 
 [[bench]]
 name = "bench"
-required-features = ["openai-vocabulary-file"]
+required-features = ["ndarray", "openai-vocabulary-file"]
```

Comment:
We should avoid this and only guard the `tokenize_batch_small` benchmark on this.

Reply:
Why is this better than just guarding the single benchmark?

Reply:
I'm not sure guarding a single benchmark is even possible in Criterion. I didn't find anything in the documentation, and tried several combinations while defining the `criterion_group!(...)` and `criterion_main!(...)` but it looks like there is no way to feature-gate individual benchmarks.

In addition, it's maybe a bit more discoverable. I tend to just run `cargo bench` on projects I'm not familiar with (instead of `cargo bench --all-features`) and would hence miss those benchmarks. By having two benchmark `.rs` files I might realize that there are actually more benchmarks I can run.